llvm/include/llvm/BinaryFormat/ELFRelocs/x86_64.def

```diff
@@ -44,3 +44,4 @@ ELF_RELOC(R_X86_64_IRELATIVE,   37)
 ELF_RELOC(R_X86_64_GOTPCRELX,   41)
 ELF_RELOC(R_X86_64_REX_GOTPCRELX,    42)
 ELF_RELOC(R_X86_64_REX2_GOTPCRELX,    43)
```

Comment:
Maybe merge https://github.com/llvm/llvm-project/pull/116737 first and rebase? So that it does not look strange.

Reply:
Okay for me.